ACE/ACE-INSTALL.html

```diff
@@ -1623,20 +1626,53 @@ <h3><a href="mailto:Jaffar_Shaikh@Mitel.COM">Jaffar Shaikh's</a>
 symbols will be created and the build will huge in size. The debug
 symbols are necessary when you want to debug your code.</p>
 
-<p></p><hr align="left" width="50%">
+<hr align="left" width="50%">
+
+<h2><a name="android">Building and Installing ACE on Android</a></h2>
+
+<ul>
+  <li><a href="#android-toolchain">Generating a Toolchain</a></li>
+  <li><a href="#android-oldndks">Using Older NDKs</a></li>
+  <li><a href="#android-building">Building</a></li>
+  <li><a href="#android-install">Installing ACE on Android</a></li>
+  <li><a href="#android-logging">Logging</a></li>
+  <li><a href="#android-openssl">OpenSSL</a></li>
+</ul>
 
-<p></p><h2><a name="android">Building and Installing ACE on Android</a></h2>
 <p>ACE can be built for Android by using the <a
 href="https://developer.android.com/ndk/">Android Native Development Kit
 (NDK)</a>. This is different than the standard way of writing Android
 applications in Java which run the on Android Runtime or the older Dalvik
 Virtual Machine. Applications built using the NDK are native Linux applications
 written in C or C++ specifically compiled to run on Android systems. In
 addition, applications built using the NDK have access to Android-specific APIs
-much like on Java-based Android applications.
+much like the ones available to Java-based Android applications.
 </p>
 
-<h3>Generating a Toolchain</h3>
+<div class="boxed">
+<p><b>Windows Users:</b> These instructions are written for a Unix based
+platform like Linux, but can also be used on Windows. If you are using an
+virtualized Linux environment like Windows Subsystem for Linux (WSL), Docker,
+or a traditional VM, then you can use the Linux version of the NDK and ignore
+rest of this note and all the other Windows specific notes.
+</p>
+
+<p>If that is not the case, you should also pay attention to the notes marked
+with "<b>Windows Users:</b>" in addition to the rest of the instructions. In
+addition to the Windows version of the Android NDK, you will also need MinGW
+for Unix utilities that ACE needs. Install <a href="http://www.mingw.org">
+MinGW</a> and the <code>mingw-developer-toolkit-bin</code> meta package using
```

Comment:
It would be good to find out what the minimal subset is, since `mingw-developer-toolkit-bin` brings a lot of dependencies in (autotools?).  I don't think we have to do it in this PR though.

Reply:
True, but I didn't want to go through them at the time and I didn't know everything that gnuace would want.

Reply:
Run procmon while it builds and then use the process tree when it's done